roles/srv-announcer/tasks/install.yml

```diff
@@ -1,47 +1,23 @@
-- name: copying {{ srv_announcer_service_name }} artifact
-  copy:
-    src: "{{ sft_artifact_file_path }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
-  when:
-    - srv_announcer_artifact_file_path is defined
-    - srv_announcer_version is not defined
-    - srv_announcer_artifact_file_url is not defined
-    - srv_announcer_artifact_checksum is not defined
-
-- name: downloading {{ srv_announcer_service_name }} artifact by version
-  get_url:
-    url: "{{ srv_announcer_download_base_url }}/{{ srv_announcer_artifact_basename }}.tar.gz"
-    checksum: "sha256:{{ srv_announcer_versions[ srv_announcer_version ][ srv_announcer_platform ~ '_' ~ srv_announcer_arch ] }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
-  when:
-    - srv_announcer_version is defined
-    - srv_announcer_artifact_file_path is not defined
-    - srv_announcer_artifact_file_url is not defined
-    - srv_announcer_artifact_checksum is not defined
-
 - name: downloading {{ srv_announcer_service_name }} artifact by URL
   get_url:
     url: "{{ srv_announcer_artifact_file_url }}"
-    checksum: "sha256:{{ srv_announcer_artifact_checksum }}"
```

Comment:
question: atm the build process only spits out SHA256, so what would be the reason to hand this over to the inventory author?

Reply:
`checksum` doesn't really imply `sha256`.

This is about allowing users to provide all other checksum methods supported by `get_url.checksum`.

Reply:
Since we are controlling the supply chain, and thus the checksum(s), it makes little sense to me to expose the whole set of capabilities that `get_url.checksum` offers.

Reply:
This discussion feels a bit bike-sheddy to me.   Having the type of hash next to the shasum is good for documentation; so I'd actually prefer personally if we keep the two together (or rename the variable to contain the hashing mechanism used).